src/plugins/rz-pm.md

```diff
@@ -1,6 +1,6 @@
 # Creating an rz-pm package of the plugin
 
-As you remember rizin has its own [packaging manager](/tools/rz-pm/intro.md) and we can easily
+As you remember rizin has its own [packaging manager](../tools/rz-pm/intro.md) and we can easily
 add newly written plugin for everyone to access.
 
 All packages are located in [rizin-pm](https://github.com/rizinorg/rizin-pm) repository, and have
```

Comment:
This one should be rewritten since `rz-pm` now has different format and options.

Reply:
this is just to fix a warning of missing file while compiling the book. once i will reach the `tools` folder i will.